models/src/main/protobuf/CasperMessage.proto

```diff
@@ -22,9 +22,10 @@ service DeployService {
   rpc addBlock(BlockMessage) returns (DeployServiceResponse) {}
   rpc createBlock(google.protobuf.Empty) returns (DeployServiceResponse) {}
   rpc showBlock(BlockQuery) returns (BlockQueryResponse) {}
-  rpc showBlocks(google.protobuf.Empty) returns (stream BlockInfoWithoutTuplespace) {}
-  rpc listenForDataAtName(Channel) returns (ListeningNameDataResponse) {}
-  rpc listenForContinuationAtName(Channels) returns (ListeningNameContinuationResponse) {}
+  rpc showMainChain(BlocksQuery) returns (stream BlockInfoWithoutTuplespace) {}
```

Comment:
I don't think this is a useful API to keep around. I know this is what `showBlocks` used to do, but that was wrong, we always should have been showing the whole dag.